ATMoviess/ATMoviess/ViewModels/ViewModelBase.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace ATMoviess.ViewModels
+{
+    public abstract class ViewModelBase : INotifyPropertyChanged
+    {
+        public async virtual Task InitializeAsync(object navigationData)
+        {
+            await Task.FromResult(false);
```

Comment:
In cases where your `async` method just runs a `Task`, you can remove the `async` notation and return the `Task` directly, so you let the caller decide whether or not it makes sense to `await` the `Task`.